<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Sugar</author>
 <comment>** Copyright (C) 2011-2017 Tycho Softworks.
**
** This program is free software: you can redistribute it and/or modify
** it under the terms of the GNU General Public License as published
** by the Free Software Foundation, either version 3 of the License, or
** (at your option) any later version.
**
** This program is distributed in the hope that it will be useful,
** but WITHOUT ANY WARRANTY; without even the implied warranty of
** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
** GNU General Public License for more details.
**
** You should have received a copy of the GNU General Public License
** along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.</comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual Page Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Manpager.qrc">
    <normaloff>:/images/manpager.png</normaloff>:/images/manpager.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="indexTab">
         <attribute name="title">
          <string>Index</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableView" name="indexView">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>16</number>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReload"/>
   <addaction name="actionClear"/>
   <addaction name="actionAbout"/>
   <addaction name="actionOptions"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Manpager.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit Manpager</string>
   </property>
   <property name="iconText">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="whatsThis">
    <string>Exit Application</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAll">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="stock_select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;All Sections</string>
   </property>
   <property name="toolTip">
    <string>All Sections</string>
   </property>
   <property name="whatsThis">
    <string>Selects all man page sections</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All &amp;But Selected</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Manpager.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Tabs</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
   <property name="whatsThis">
    <string>Clears all man section selections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSection1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;1: User Commands</string>
   </property>
   <property name="iconText">
    <string>&amp;1: User Commands</string>
   </property>
   <property name="toolTip">
    <string>1: User Commands</string>
   </property>
   <property name="whatsThis">
    <string>Select user command pages</string>
   </property>
  </action>
  <action name="actionSection2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;2: System Calls</string>
   </property>
   <property name="toolTip">
    <string>2: System Calls</string>
   </property>
   <property name="whatsThis">
    <string>Select system call pages</string>
   </property>
  </action>
  <action name="actionSection3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;3: Subroutines</string>
   </property>
   <property name="iconText">
    <string>3: Subroutines</string>
   </property>
   <property name="toolTip">
    <string>3: Subroutines</string>
   </property>
   <property name="whatsThis">
    <string>Select subroutine pages</string>
   </property>
  </action>
  <action name="actionSection4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;4: Devices</string>
   </property>
   <property name="toolTip">
    <string>4: Devices</string>
   </property>
   <property name="whatsThis">
    <string>Select device pages</string>
   </property>
  </action>
  <action name="actionSection5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;5: File Formats</string>
   </property>
   <property name="toolTip">
    <string>5: File Formats</string>
   </property>
   <property name="whatsThis">
    <string>Select file format pages</string>
   </property>
  </action>
  <action name="actionSection6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;6: Games</string>
   </property>
   <property name="toolTip">
    <string>6: Games</string>
   </property>
   <property name="whatsThis">
    <string>Select games pages</string>
   </property>
  </action>
  <action name="actionSection7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;7: Miscellaneous</string>
   </property>
   <property name="toolTip">
    <string>7: Miscellaneous</string>
   </property>
   <property name="whatsThis">
    <string>Select miscellaneous pages</string>
   </property>
  </action>
  <action name="actionSection8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;8: Administration</string>
   </property>
   <property name="toolTip">
    <string>8: Administration</string>
   </property>
   <property name="whatsThis">
    <string>Select administration pages</string>
   </property>
  </action>
  <action name="actionSectionl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;l: Local</string>
   </property>
   <property name="toolTip">
    <string>l: Local</string>
   </property>
   <property name="whatsThis">
    <string>Select locally installed pages</string>
   </property>
  </action>
  <action name="actionSectionn">
   <property name="text">
    <string>&amp;n: New</string>
   </property>
   <property name="toolTip">
    <string>n: New</string>
   </property>
   <property name="whatsThis">
    <string>Select new pages</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Index Search</string>
   </property>
  </action>
  <action name="actionKeywords">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keyword Search</string>
   </property>
  </action>
  <action name="action_Mode">
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="toolTip">
    <string>Change Search Mode</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="Manpager.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload Index</string>
   </property>
   <property name="iconText">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload Index</string>
   </property>
   <property name="whatsThis">
    <string>Reload manual sections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Manpager.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Manpager</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>Copyright and author information</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSupport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/web/github.png</normaloff>:/icons/web/github.png</iconset>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
   <property name="toolTip">
    <string>Support</string>
   </property>
   <property name="whatsThis">
    <string>Access to support and source code</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="Manpager.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="iconText">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View in Tab</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Manpager.qrc"/>
 </resources>
 <connections/>
</ui>
